### DISCUSSION TITLE
Quickstart error - Unknown error response code 403

### DISCUSSION THREAD

**Discussion by MattiaCandal** [iTwin/itwinjs-core#4543] [General] [answered]
Hi to all, I am writing to you because I am struggling in getting the quickstart tutorial work for me :(.

As I have seen from previous threads, other users had similar problems but I cannot manage to correctly start my test. I am attaching a screenshot with x-correlation id since it was frequently requested to understand the problem in other cases. I am a novice regarding programming so I don't properly know what I am doing and what could be wrong. I followed the quickstart step by step. The error as far as I know should be related to server response, being a 403. But I don't know much more about it.

![image](https://user-images.githubusercontent.com/114148193/197788222-1db68564-d179-4e69-830a-9d7f67c08445.png)

l. Moreover when i run the npx command to create the react app I got a lot of warnings about dependencies (e.g. deprecated, different versions, vulnerabilities etc), I don't know it is ok or if it is the underlying problem.

If anyone could help me I would be really gratefu

**Comment by aruniverse** [comment, marked as answer]
Hi,

Can you try again? We unfortunately had to momentarily roll back the version deployed to the imodel-access service to fix an issue, but a fix has just been deployed, so everything should be working again now. 

Regarding the warnings you're seeing, can you share what those are?

**Comment by MattiaCandal** [comment]
Hi Aruniverse. I tried to start now and it goes correctly now. I am still laughing because not only I am a newbie which does not know what he was doing properly, but I got the wrong day for doing it :D ahahaha

![image](https://user-images.githubusercontent.com/114148193/197805539-ff77316a-1558-4e29-a707-593f94ef6577.png)

I am sending to you a screenshot preview, but I am also attaching a word file with the whole process copy-pasted for further information.
Should I do npm audit fix or something else? I am concerned with critical vulnerabilities in particular. Thank you
[Log_warn deprecated.docx](https://github.com/iTwin/itwinjs-core/files/9861314/Log_warn.deprecated.docx)

**Reply by aruniverse** [reply]
You should be able to safely ignore the peer dep of `react-data-grid` missing. Re the deprecated packages warnings, I believe most of them are coming from transitive dependencies outside of our control. 

Re the security vulnerabilities, it would be a good idea for you to look over the critical ones, but most of them are probably irrelevant. If you get a chance I'd recommend reading https://overreacted.io/npm-audit-broken-by-design/ ; welcome to the npm ecosystem :)

With respect to using iTwin.js, you should be ok, and the warnings you're seeing in the console can mostly all be ignored. If you run into any other issues keep us posted and we'd be more than happy to help.

**Reply by MattiaCandal** [reply]
Thank you Aruniverse, I was concerned about this because in the previous month I installed all the stuff such as npm, conda etc and I thought that maybe I made a mess with previous installed versions of other projects leftovers. In fact, I already knew the article you linked but I never had the opportunity to discuss it with an expert, so thank you for your time too.
